code/scripts/robot.py

```diff
@@ -0,0 +1,7 @@
+import math
+import random
+
+
+def test():
+    for i in range(0, 5):
+        print(math.sqrt(random.randint(10 * i, 10 * (i + 1))))
```

Comment:
did you test calling a controller function, like move?

Reply:
Yeah, you can run

```
import emb
emb.move(1, 5)
```
with the simulator window open